.concourse/tasks/upgrade.sh

```diff
@@ -42,13 +41,9 @@ KUBECF_LATEST_RELEASE="$(cat kubecf-github-release/version)"
 export KUBECF_LATEST_RELEASE
 export SCF_CHART="https://github.com/cloudfoundry-incubator/kubecf/releases/download/v${KUBECF_LATEST_RELEASE}/kubecf-bundle-v${KUBECF_LATEST_RELEASE}.tgz"
 
-export ENABLE_EIRINI=false
```

Comment:
I think being explicit is good in this case (so people don't wonder what the default is)

Reply:
Fixed